android/quest/src/main/java/org/smartregister/fhircore/quest/ui/main/AppMainViewModel.kt

```diff
@@ -313,6 +314,12 @@ constructor(
       ExistingPeriodicWorkPolicy.REPLACE,
       PeriodicWorkRequestBuilder<FhirTaskPlanWorker>(12, TimeUnit.HOURS).build()
     )
+    // Schedule job for generating measure report in the background
+    workManager.enqueueUniquePeriodicWork(
+      MeasureReportWorker.WORK_ID,
+      ExistingPeriodicWorkPolicy.KEEP,
+      PeriodicWorkRequestBuilder<MeasureReportWorker>(24, TimeUnit.HOURS).build()
```

Comment:
Can we configure the 24hr duration via `ApplicationConfiguration`?

We may need a model for period configuration where we can provide the `interval`, and time unit (DAYS, HOURS, MINUTES...etc)

```kotlin 
data class RepeatIntervalConfig(val interval: Int = 15, val timeUnit: TimeUnit)
```
then create 3 new properties in `ApplicationConfiguration`

```kotlin
taskUpdateInterval: RepeatIntervalConfig = RepeatIntervalConfig(12, TimeUnit.HOURS)
syncInterval: RepeatIntervalConfig = RepeatIntervalConfig()
reportGenerationInterval: RepeatIntervalConfig = RepeatIntervalConfig(24, TimeUnit.HOURS)
```